docs/vsa/vsa.rst

```diff
@@ -0,0 +1,21 @@
+.. doctest-skip-all
+
+.. _astroquery.vsa:
+
+******************************
+VSA Queries (`astroquery.vsa`)
+******************************
+
+The Vista Science Archive (VSA; http://horus.roe.ac.uk/vsa/index.html) shares a
+common interface with the :doc:`../ukidss/ukidss` archive.  Both are hosted
+by the Wide Field Astronomy Unit (WFAU; www.roe.ac.uk/ifa/wfau;
+:doc:`../wfau/wfau`).  Please see the UKIDSS documentation pages for
+instructions and examples.
```

Comment:
In the long term it would be nice to have some vista specific examples here, but that can be done in a follow-up PR.